tests/chainer_tests/test_link.py

```diff
@@ -98,6 +98,18 @@ def test_to_gpu(self):
         self.assertIsInstance(self.link.y.grad, cupy.ndarray)
         self.assertIsInstance(self.link.p, cupy.ndarray)
 
+    @attr.gpu
+    def test_to_gpu_different_device(self):
+        cuda.Device(1).use()
```

Comment:
I'm afraid I'm not sure what you mean, but I added tearDown method to reset the current device.